custom_mutators/symcc/symcc.c

```diff
@@ -135,6 +141,7 @@ void afl_custom_queue_new_entry(my_mutator_t * data,
 
         ssize_t r = read(fd, data->mutator_buf, MAX_FILE);
         DBG("fn=%s, fd=%d, size=%ld\n", fn, fd, r);
+        ck_free(fn);
         if (r <= 0) return;
         close(fd);
```

Comment:
better move close(fd) one line up before the return to avoid leaking fd.

Reply:
good catch!

Reply:
If we can get this fix as well, I am happy.